docs/pages/project/proposal.md

```diff
@@ -0,0 +1,79 @@
+# Meshery Proposal for CNCF Incubation
+
+## About Meshery
+
+[Meshery](https://meshery.io/) is a cloud native management plane designed to facilitate the management, configuration, and performance testing of cloud native in cloud-native environments. 
```

Comment:
1. I see word `cloud native` 3 times in single sentence. Please see if we can modify sentence a bit.
2. Please use either  `cloud native` or `cloud-native`. Seems like `cloud native` is common in CNCF space

Reply:
good call, @vishwasrao 👍